grinder/core.py

```diff
@@ -1367,10 +1377,7 @@ def masscan_scan(
         except Exception as masscan_exception:
             if "network is unreachable" in str(masscan_exception):
```

Comment:
Same here: why we catch generic `Exception` and look for a substring in the message? Masscan doesn't have its own exception handlers?

Reply:
Fixed